### ISSUE TITLE
Änderung in "Abfülldaten" -> "Karbonisierung" -> "Süßkraft des Zuckers" ist global

### ISSUE THREAD

**Issue by markussoeth** [kleiner-brauhelfer/kleiner-brauhelfer-2#282]
Hallo liebe Programmierer,

mir ist eben aufgefallen:
Ändere ich bei einem Rezept die Süßkraft des Zuckers, schlägt sich der Eintrag auf alle Rezepte durch.
Auch auf die schon abgefüllten und die noch nicht abgefüllten.

Ich musste ich mal machen, da ich Vollrohrzucker in einem Rezept genutzt habe, der nur einen Zuckergehalt von 92% hatte.

Es wäre schön, wenn der Eintrag auf das Rezept bezogen bleibt.

**Comment by Rasinari** [comment]
Stimmt. Es bezieht sich auf alle ungebrauten Rezepte. Ist definitiv ein Bug.

Workaround derzeit: Rezept mit Zuckergehalt abfüllen und speichern. Dann beim nächsten Rezept Zuckergehalt umstellen. kann bis vors Umfüllen geändert werden, sollte man aber nicht übersehen (mir schon passiert).

**Comment by markussoeth** [comment]
Der Fehler bezieht sich auch auf schon gebraute und abgefüllte Rezepte.
Wenn man es weis nicht soo schlimm. Nur bei mir bestimmt eine zukünftige Fehlerquelle :-)

Hab jetzt das Stichwort Bug hinzugefügt.

**Comment by BourgeoisLab** [comment]
Das ist gleich, wie z.B. die Flaschengrösse oder den Umrechnungsfaktor für den Refraktometer. Das wird alles nicht in der Datenbank gespeichert, sondern sind Einstellungen. Aus meiner Sicht also kein Bug, sondern einfach anders implementiert:).

Ich denke nicht, dass ich das ändern werde. Das bläht die Datenbank nur auf. Man muss es einfach wissen;)

**Comment by Rasinari** [comment]
> Das ist gleich, wie z.B. die Flaschengrösse oder den Umrechnungsfaktor für den Refraktometer. Das wird alles nicht in der Datenbank gespeichert, sondern sind Einstellungen. Aus meiner Sicht also kein Bug, sondern einfach anders implementiert:).
> 
> Ich denke nicht, dass ich das ändern werde. Das bläht die Datenbank nur auf. Man muss es einfach wissen;)

Entschuldigung, ich muss hier widersprechen. Was Markus prophezeit, ist mir schon passiert. Ich habe einfach übersehen, dass ich bei einer anderen Rezeptentwicklung Haushaltszucker eingestellt hatte, dabei sich aber vor dem Abfüllen ein Bier mit Traubenzucker auch verändert hatte. Folge war eine zu geringe Karbonisierung.

Insbesondere wenn man wie ich regelmäßig sechs und mehr Biere in der Pipeline hat, ist dass die Atz oder Wildtierfütterung schlechthin. Ob man es jetzt Bug nennt oder nicht, ist unerheblich. Es produziert Fehler.

Bitte unbedingt ändern, damit man pro Sud entscheiden kann! Danke!
